### PR TITLE
Fix websocket exchanges

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_trading/exchanges/types/websocket_exchange.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def on_pong(self):
 
     async def on_ping(self):
         self.logger.debug("Ping received. Sending pong...")
-        self.websocket.pong()
+        await self.websocket.pong()
```
Comment (Herklos) [inline octobot_trading/exchanges/types/websocket_exchange.py:181]: ?

File: octobot_trading/exchanges/rest_exchange.py [hunk 2]
```diff
@@ -464,7 +464,7 @@ def parse_ticker(self, ticker):
         return self.client.parse_ticker(ticker)
 
     def parse_ohlcv(self, ohlcv):
-        return self.client.parse_ohlcv(ohlcv)
+        return self.exchange_manager.uniformize_candles_if_necessary(self.client.parse_ohlcv(ohlcv))
```
Comment (Herklos) [inline octobot_trading/exchanges/rest_exchange.py:467]: :+1:

File: octobot_trading/exchanges/types/websocket_exchange.py (hunk 1 above)
Comment (GuillaumeDSM) [reply octobot_trading/exchanges/types/websocket_exchange.py:181]: Ping and pong methods are async https://websockets.readthedocs.io/en/stable/api.html#websockets.protocol.WebSocketCommonProtocol.ping

Comment (Herklos) [reply octobot_trading/exchanges/types/websocket_exchange.py:181]: Yes my bad 👍

**Pull request merged**